v1/examples/charge/main.go

```diff
@@ -9,17 +9,14 @@ import (
 func main() {
 	pay := payjp.New("sk_test_c62fade9d045b54cd76d7036", nil)
 
+	var cardToken string = "生成したトークン"
+
 	// 支払いをします
 	charge, _ := pay.Charge.Create(3500, payjp.Charge{
 		// 現在はjpyのみサポート
 		Currency: "jpy",
 		// カード情報、顧客ID、カードトークンのいずれかを指定
```

Comment:
カード情報はCardって引数だけど通過型なのでダメです、消しましょー。
顧客IDはCustomerIDって別引数なので消しましょう。

```suggestion
		// カードトークンを指定（サンプルのトークンは以下などで生成できます）
		// https://pay.jp/docs/checkout
```

sk_test_xxxに対応する公開鍵のリンクも載せてます。これをしないなら、そもそもsk_test_xxxもダミーの値かyour_sk_keyとかに変えた方が良いかと